inspirehep/submissions/views.py

```diff
@@ -165,6 +166,23 @@ def start_workflow_for_submission(self, submission_data, control_number=None):
         abort(503)
 
 
+@shared_task(ingore_result=False, max_retries=5)
+def async_create_ticket_with_template(
```

Comment:
I'd move this to ``tasks.py``

Reply:
I don't mind but shall we keep this, until we need it somewhere else?

Reply:
We always have the tasks in ``tasks.py`` for consistency let's move this one as well.